src/OMSimulatorLib/TLMBusConnector.cpp

```diff
@@ -260,6 +305,48 @@ oms3::Component *oms3::TLMBusConnector::getComponent()
   return component;
 }
 
+oms3::TLMBusConnector* oms3::TLMBusConnector::getActualBus()
+{
+  if(!actualBus && !parentSystem)
+    actualBus = this;
+  else if(!actualBus)
+    actualBus = getActualBus(getName(), parentSystem);
+
+  return actualBus;
+}
+
+oms3::TLMBusConnector* oms3::TLMBusConnector::getActualBus(ComRef cref, System *system)
+{
+  Connection** connections = system->getConnections(ComRef(""));
```

Comment:
Why do you call `system->getConnections` with an empty `ComRef` as argument?

Reply:
Because I want the system's own connections. The ComRef argument is mandatory, and it needs to be empty since I don't want connections for a subsystem.